docs/src/main/sphinx/security/password-file.rst

```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
 
     htpasswd -B -C 10 password.db test
 
+.. _verify_authentication:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify password file authentication, log in to the :doc:`Web UI
+</admin/web-interface>`, and connect with the Trino :doc:`CLI </client/cli>` to
+the cluster.
+
+* Connect to the Web UI from your browser using a URL that uses HTTPS, such as
+  ``https://trino.example.com:8443``. Enter a username in the ``Username`` text
+  box and the corresponding password in the ``Password`` text box, and log in to
+  the UI. Confirm that you are not able to log in using an incorrect username
+  and password combination.
```

Comment:
Add that the username is shown in the top right corner after logging in successfully.